lib/reverseString.js

```diff
@@ -0,0 +1,4 @@
+const reverseString = (input) => {
+    return input.split("").reverse().join("");
```

Comment:
👍 factoring out utility functions
You can eliminate the brackets here but just a stylistic choice